docs/supportedHashes.json

```diff
@@ -7,6 +7,10 @@
         "name": "PAL GC",
         "sha1": "0227d7c0074f2d0ac935631990da8ec5914597b4"
     },
+    {
+        "name": "PAL GC MQ",
```

Comment:
for the debug ones we just use `PAL MQ`, this is because MQ was only ever released on GC, so i figured GC MQ is redundant. i think it'd be nice to just use `PAL MQ` for consistency.